src/calendar/day/marking/index.tsx

```diff
@@ -45,14 +50,16 @@ export interface MarkingProps extends DotProps {
   startingDay?: boolean;
   endingDay?: boolean;
   accessibilityLabel?: string;
+  customStyles?: CustomStyle;
```

Comment:
What is new type? 
I don't we're using it in this file..

Reply:
On line #17 I declare the CustomStyle type